### DISCUSSION TITLE
blog/advanced-typescript

### DISCUSSION THREAD

**Discussion by giscus** [angular-experts-io/angular-experts-website-comments#5] [General]
# blog/advanced-typescript

Get familiar with some of Typescript's greatest advanced features.

https://angularexperts.io/blog/advanced-typescript

**Comment by CameronJGuthrie** [comment]
Thanks for the article, it was a good read!

I think you've provided an example for a getters util type instead of setters, but I'm still amazed at what can be done with typescript these days.
